### ISSUE TITLE
fix: TypeError: Cannot read property 'lighthouse_accessibility' of undefined

### ISSUE THREAD

**Issue by tristanrobert** [SocialGouv/dashlord-actions#6]
yarn start:

`dataGetter
src/components/Dashboard.tsx:104
  101 | const summary = (rowData as UrlReport).summary;
  102 | const scoreKey = `lighthouse_${id}`;
  103 | //@ts-expect-error
> 104 | if (summary[scoreKey] === undefined) {
      | ^  105 |   return -1;
  106 | }
  107 | //@ts-expect-error`

**Comment by tristanrobert** [comment]
il manque les summary dans `report.json`

**Comment by tristanrobert** [comment]
une regénération du report devrait suffire

**Comment by tristanrobert** [comment]
Fixed in PR #7
